packages/components/src/checkbox-control/style.scss

```diff
@@ -57,7 +57,7 @@ svg.dashicon.components-checkbox-control__checked {
 	pointer-events: none;
 }
 
-@media screen and ( max-width: 728px ) {
+@media screen and ( max-width: 600px ) {
```

Comment:
Do you think it's possible to replace this media query without our media query mixins?